dsp/modules/lm.py

```diff
@@ -60,6 +60,7 @@ def inspect_history(self, n: int = 1, skip: int = 0):
                     "Sagemaker",
                     "premai",
                     "tensorrt_llm",
+                    "unify",
```

Comment:
should we leave this line?

Reply:
I'd say so- otherwise line 107 would probably have to be edited.